src/platform/julius.c

```diff
@@ -18,6 +18,7 @@
 #include "tinyfiledialogs/tinyfiledialogs.h"
 
 #include <signal.h>
+#include <stddef.h>
```

Comment:
Include is no longer necessary.

Reply:
I swear I had removed it, thanks for noticing.